openpype/hosts/nuke/plugins/publish/extract_review_data_mov.py

```diff
@@ -50,51 +52,33 @@ def process(self, instance):
                 f_task_types = o_data["filter"]["task_types"]
                 f_subsets = o_data["filter"]["sebsets"]
 
+                self.log.debug(
+                    "f_families `{}` > families: {}".format(
+                        f_families, families))
+
+                self.log.debug(
+                    "f_task_types `{}` > task_type: {}".format(
+                        f_task_types, task_type))
+
+                self.log.debug(
+                    "f_subsets `{}` > subset: {}".format(
+                        f_subsets, subset))
+
                 # test if family found in context
-                test_families = any([
-                    # first if exact family set is matching
-                    # make sure only interesetion of list is correct
-                    bool(set(families).intersection(f_families)),
-                    # and if famiies are set at all
-                    # if not then return True because we want this preset
-                    # to be active if nothig is set
-                    bool(not f_families)
-                ])
+                # using intersection to make sure all defined
+                # families are present in combination
+                if f_families and not any(
+                        set(families).intersection(f_families)):
```

Comment:
No need to check `any` because if the set intersection is empty it will already be used as `False`.
Thus you should be able to just do:
```
               if f_families and not set(families).intersection(f_families):
```

Even better if you change `families` into a set outside of the for loop at the top:

```
families = set(instance.data["families"])
families.add(instance.data["family"])
```

And then here just do:
```
               if f_families and not families.intersection(f_families):
```

Reply:
nice, thanks